bridge/engine/ActiveMediaList.cpp

```diff
@@ -646,7 +647,7 @@ bool ActiveMediaList::makeUserMediaMapMessage(const size_t lastN,
         const auto rewriteMapItr = _videoSsrcRewriteMap.find(videoListEntry->_data);
```

Comment:
This is incorrect. The UMM for clients must also check the video stream endpoints in the barbells.
Unless we add fake video streams there due to UMM over the barbell.